### DISCUSSION TITLE
Error on upgrading the Microsoft.WindowsAppSDK to latest version (1.2.230118.102)

### DISCUSSION THREAD

**Discussion by NaveenBadal** [unoplatform/uno#11088] [Q&A]
When trying to upgrade the common project (.shared one) from 1.1.3 (which comes by default on creating) to the latest version 1.2.230118.102 or 1.2.221209.1, Error is coming stating that "Duplicate Content Items were included". 

I am using the .Net 7 and VS 2022 latest preview.
![Screenshot 2023-01-28 123605](https://user-images.githubusercontent.com/42309324/215254956-1381ce3d-3b9b-49b2-932b-2bbbf2a8a0de.png)

Because of this, prjoject is not getting build.
Please see the attached screenshot.

**Comment by jeromelaban** [comment]
Thanks for the feedback!

Can you specify which part of the app (which project, specifically) you are upgrading?
